### ISSUE TITLE
Incompatible PKL's?

### ISSUE THREAD

**Issue by atugar** [mit-han-lab/data-efficient-gans#12]
Hi. Is the PKL generated by run_few_shot not compatible with regular StyleGAN2? I tried to transfer learn from a PKL generated by run_few_shot using a regular StyleGAN2 repo and got the following error:

```
Building TensorFlow graph...
Traceback (most recent call last):
  File "/tensorflow-1.15.2/python3.6/tensorflow_core/python/framework/ops.py", line 642, in set_shape
    unknown_shape)
tensorflow.python.framework.errors_impl.InvalidArgumentError: Shapes must be equal rank, but are 2 and 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/run_training.py", line 192, in <module>
    main()
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/run_training.py", line 187, in main
    run(**vars(args))
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/run_training.py", line 120, in run
    dnnlib.submit_run(**kwargs)
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/dnnlib/submission/submit.py", line 343, in submit_run
    return farm.submit(submit_config, host_run_dir)
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/dnnlib/submission/internal/local.py", line 22, in submit
    return run_wrapper(submit_config)
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/dnnlib/submission/submit.py", line 280, in run_wrapper
    run_func_obj(**submit_config.run_func_kwargs)
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/training/training_loop.py", line 220, in training_loop
    G_loss, G_reg = dnnlib.util.call_func_by_name(G=G_gpu, D=D_gpu, opt=G_opt, training_set=training_set, minibatch_size=minibatch_gpu_in, **G_loss_args)
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/dnnlib/util.py", line 256, in call_func_by_name
    return func_obj(*args, **kwargs)
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/training/loss.py", line 152, in G_logistic_ns_pathreg
    fake_images_out, fake_dlatents_out = G.get_output_for(latents, labels, is_training=True, return_dlatents=True)
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/dnnlib/tflib/network.py", line 221, in get_output_for
    out_expr = self._build_func(*final_inputs, **build_kwargs)
  File "<string>", line 206, in G_main
  File "/content/drive/My Drive/Nvidia_Stylegan2/stylegan2/dnnlib/tflib/network.py", line 221, in get_output_for
    out_expr = self._build_func(*final_inputs, **build_kwargs)
  File "<string>", line 281, in G_mapping
  File "/tensorflow-1.15.2/python3.6/tensorflow_core/python/framework/ops.py", line 645, in set_shape
    raise ValueError(str(e))
ValueError: Shapes must be equal rank, but are 2 and 1
```

**Comment by zsyzzsoft** [comment]
Not supported. It would be better to continue training using this repo as well.

**Comment by zsyzzsoft** [comment]
You may also try the `resume_with_new_nets` option of the `training_loop` function, if you can ensure that the network hyperparameters are the same.

**Comment by YotamNitzan** [comment]
Hi!
Is the other way around not compatible too? I would like to use an FFHQ snapshot as pre-training before using your method.
Currently just using the --resume flag throws this error:

```
Traceback (most recent call last):
  File "/disk1/anaconda3/envs/sg2-ada-torch/lib/python3.7/site-packages/torch/multiprocessing/spawn.py", line 19, in _wrap
    fn(i, *args)
  File "/disk1/projects/data-efficient-gans/DiffAugment-stylegan2-pytorch/train.py", line 398, in subprocess_fn
    training_loop.training_loop(rank=rank, **args)
  File "/disk1/projects/data-efficient-gans/DiffAugment-stylegan2-pytorch/training/training_loop.py", line 160, in training_loop
    misc.copy_params_and_buffers(resume_data[name], module, require_all=False)
  File "/disk1/projects/data-efficient-gans/DiffAugment-stylegan2-pytorch/torch_utils/misc.py", line 160, in copy_params_and_buffers
    tensor.copy_(src_tensors[name].detach()).requires_grad_(tensor.requires_grad)
RuntimeError: The size of tensor a (512) must match the size of tensor b (256) at non-singleton dimension 0
```

Is there another way? How would you suggest using pre-training?

**Comment by YotamNitzan** [comment]
I think I solved it by changing fmap to 0.5 like it should be for 256 models.
